beacon-chain/core/helpers/validators_test.go

```diff
@@ -144,7 +144,7 @@ func TestIsSlashableValidator_InactiveSlashed(t *testing.T) {
 func TestBeaconProposerIndex_OK(t *testing.T) {
 	ClearAllCaches()
 
-	if params.BeaconConfig().SlotsPerEpoch != 64 {
+	if params.BeaconConfig().SlotsPerEpoch != 32 {
 		t.Errorf("SlotsPerEpoch should be 64 for these tests to pass")
```

Comment:
Delete these assertions if you could change this number and no tests fail. We had this because we wanted to fail faster on config changes